js/templates/helpers/icon.js

```diff
@@ -1,4 +1,4 @@
-define(['handlebars'], function(Handlebars) {
+define(['jquery', 'handlebars'], function($, Handlebars) {
```

Comment:
I think we removed jQuery as a global when we started the Vue.js migration.
So it was only broken on the dev branch (discovered when testing issue modal on dataset page)